### PR TITLE
feat: add cronjob to `employment-hero-app` chart

### PR DESCRIPTION
**Pull request by suonlight**
This PR is to add cronjob config to employment-hero-app. 

Currently we use concurrencyPolicy=Forbid. But in my opinion, we need to review the cronjob implementation by development teams and monitor to determine what concurrencyPolicy is suitable for each cronjob implementation.

### CODE REVIEW

File: employment-hero-app/templates/cronjob.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  successfulJobsHistoryLimit: {{ default 3 $.Values.config.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ default 3 $.Values.config.failedJobsHistoryLimit }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: {{ $cron_job_app_name }}
+            {{- include "image_labels" $ | indent 12 }}
+            {{- include "command_labels" (index $job "command") | indent 12 }}
```
Comment (longseespace) [inline employment-hero-app/templates/cronjob.yaml]: Add other configurations for containers within CronJob as well, such as: `env`, `resources` and `secret files`. See `web-deployment.yml` for references

**Comment by suonlight** [comment]
@longseespace please review my update.

File: employment-hero-app/templates/cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    spec:
+      template:
+        spec:
+          containers:
+          - name: {{ $cron_job_app_name }}
+            {{- if $.Values.files }}
+            volumeMounts:
+            - name: "kubestore"
+              mountPath: "/etc/kubestore"
+              readOnly: true
```
Comment (longseespace) [inline employment-hero-app/templates/cronjob.yaml]: This is incomplete. You'll need to declare `volumes` as well, not just `volumeMounts`. In the same level as `containers:`, add this:

```
volumes:
- name: "kubestore"
  secret:
    secretName: "{{ .Values.app.name }}-files"
```

File: employment-hero-app/templates/cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{{- range $job_name, $job := .Values.app.cron_jobs }}
+{{- $cron_job_app_name := (printf "%s-cronjob-%s" $.Values.app.name $job_name) | snakecase | replace "_" "-" -}}
```
Comment (suonlight) [inline employment-hero-app/templates/cronjob.yaml]: The `metadata.name` is not allowed underscore. The error is not exist on my computer. Let me see

File: employment-hero-app/templates/cronjob.yaml (hunk 2 above)
Comment (suonlight) [inline employment-hero-app/templates/cronjob.yaml]: 👍

File: employment-hero-app/templates/cronjob.yaml (hunk 3 above)
Comment (longseespace) [inline employment-hero-app/templates/cronjob.yaml]: Hmm maybe you're using a newer version of helm?

**Pull request merged**